server/prisma/legacy-data/events.yaml

```diff
@@ -0,0 +1,252 @@
+- eventid: 1
+  seasonid_fk: 1
+  eventname: Our Town
+  eventdescription: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+  active: false
+  seasonticketeligible: true
+  imageurl: https://www.arts.gov/sites/default/files/styles/portrait_crop/public/Our%20Town%20Cover%20new.jpg?h=52605a11&itok=PE1XH64E
+- eventid: 2
```

Comment:
Is there any reason for saving older versions of `event.yaml` and `donations.yaml` in a new `legacy-data` folder? I think we could probably remove these and keep the ones we have updated in the `yaml-seeder-data` folder